react/features/app/middlewares.native.js

```diff
@@ -13,6 +13,7 @@ import '../mobile/proximity/middleware';
 import '../mobile/wake-lock/middleware';
 import '../mobile/watchos/middleware';
 import '../share-room/middleware';
+import '../shared-video/middleware';
 import '../youtube-player/middleware';
```

Comment:
This one needs to go, right?

Reply:
I thought about this, but wouldn't it be safer if we do this after we remove the whole youtube-player feature from the project structure?

Reply:
What do we have still there? Can't we move it to shared-video ?

Reply:
We have some styles and a YoutubeLargeVideo component that is used in ParticipantView.native.js.

Reply:
Move that to the native components section of shared-video and fix the imports please

Reply:
Done.